docker/Dockerfile

```diff
@@ -98,11 +99,15 @@ ENV MINIFI_VERSION ${MINIFI_VERSION}
 
 # Setup minificpp user
 RUN addgroup -g ${GID} ${USER} && adduser -u ${UID} -D -G ${USER} -g "" ${USER} && \
-    install -d -o ${USER} -g ${USER} ${MINIFI_BASE_DIR}
+    install -d -o ${USER} -g ${USER} ${MINIFI_BASE_DIR} && \
+    ln -s $(which ccache) /usr/local/bin/gcc && \
+    ln -s $(which ccache) /usr/local/bin/g++
```

Comment:
This shouldn't be needed. Our CMakeLists.txt (line 141) is looking for ccache and using it if available, even if the compiler binaries are not symlinks to ccache itself.

Reply:
Updated in 34bc56c2cbef5932e2025ea09eec6f5b906af595